portfolio/src/main/webapp/script.js

```diff
@@ -30,13 +30,27 @@ function addRandomGreeting() {
 /**
  * Fetches a welcome message from the server and adds it to the DOM.
  */
-function getWelcomeMessage() {
+function getComments(selectObject) {
+  var lang = selectObject.value;
 
+  var english = "en";
+  var french = "fr";
+  var spanish = "es"
+  
   // The fetch() function returns a Promise because the request is asynchronous.
-  const responsePromise = fetch('/data');
-
+  if(lang == english){
```

Comment:
What do you think about using a switch statement instead?

more info about switch vs if-else: https://www.oreilly.com/library/view/high-performance-javascript/9781449382308/ch04.html#if-else_versus_switch

Reply:
I tried and for some reason it doesn't display the comments anymore. I will continue looking into this because I agree that i could be useful if I want to implement more languages but I will keep the if/else format for now